base/markdown/Common/inline.jl

```diff
@@ -106,6 +106,42 @@ function footnote_link(stream::IO, md::MD)
     end
 end
 
+@trigger '<' ->
+function autolink(stream::IO, md::MD)
+    withstream(stream) do
+        startswith(stream, '<') || return
+        url = readuntil(stream, '>')
```

Comment:
I think `readuntil` returns `nothing` if the input doesn't have '>'. Try with e.g. `Markdown.parse("<link")`. Try to add  `url ≡ nothing && return`